modules/quadkey/bigquery/sql/ST_ASQUADINT_POLYFILL.sql

```diff
@@ -17,6 +17,22 @@ AS """
     return quadints.map(String);
 """;
 
+CREATE OR REPLACE FUNCTION `@@BQ_PREFIX@@quadkey.__geometrycollection`
+(geojson STRING)
+RETURNS STRING
+DETERMINISTIC
+LANGUAGE js
+OPTIONS (library=["@@BQ_LIBRARY_BUCKET@@"])
+AS """
+    const pol = JSON.parse(geojson);
+    return JSON.stringify(quadkeyLib.geometrycollection(pol))
+    // Iterate through collection features
```

Comment:
Why are all the comments here?

Reply:
It was early WIP, just in case Alex had to continue. The function works now, just have to add integration tests.

By the way @alex-graciano I have seen that we have the same problem with H3 polyfill. Do you want to tackle that one? It's not crashing but it returns an empty array.